winter/http/response_header_resolver.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework.request import Request
+
+from winter.http.response_header import _BaseResponseHeader
+from winter.type_utils import is_origin_type_subclasses
+from .response_header import ResponseHeaderAnnotation
+from .response_header import ResponseHeaders
+from .. import ArgumentResolver
+from ..core import ComponentMethodArgument
+
+
+class ResponseHeaderArgumentResolver(ArgumentResolver):
+
+    def is_supported(self, argument: ComponentMethodArgument) -> bool:
+        if not is_origin_type_subclasses(argument.type_, _BaseResponseHeader):
```

Comment:
it seems just inspect.isclass(argument.type_) and issubclass(argument.type_, check_class) - enough